static/src/stylesheets/module/content/_photo-essay.scss

```diff
@@ -0,0 +1,93 @@
+.content--article {
+    .element-image--photo-esssay {
+        padding-bottom: 0;
+        margin-bottom: 0.5rem;
+
+        &.fig--no-caption.fig--has-shares {
+            padding-bottom: 0px !important;
+        }
+
+        .block-share {
+            display: none !important;
```

Comment:
We have the `is-hidden` class in `_helpers.scss`; could adding the class to the element work instead?
```
.is-hidden,
[hidden] {
    display: none !important;
}
```